config/kubernetes/sig-release/teams.yaml

```diff
@@ -94,31 +94,86 @@ teams:
     - xmudrii # 1.15 Bug Triage Shadow
     - zacharysarah # Docs
     privacy: closed
-  kubernetes-release-managers:
-    description: People actively working on next k8s minor release.  Gives admin access to repos
-      where branches must be created, etc., and write access to ones where label/PR
-      management is needed. Remove users who are not actively doing this job.
+  licensing:
+    description: Members of the Licensing subproject.
+    maintainers:
+    - nikhita # subproject owner
+    members:
+    - dims # subproject owner
+    - justaugustus # subproject owner
+    - swinslow # subproject owner
+    privacy: closed
+  release-engineering:
+    description: Members of the Release Engineering subproject.
+    maintainers:
+    - calebamiles # subproject owner
     members:
+    - aleksandra-malinowska # subproject owner
+    - hoegaarden # subproject owner
+    - ixdy # subproject owner
+    - tpepper # subproject owner
+    privacy: closed
+    teams:
+      kubernetes-release-managers:
+        description: People actively working on next k8s minor release.  Gives admin access to repos
+          where branches must be created, etc., and write access to ones where label/PR
+          management is needed. Remove users who are not actively doing this job.
+        members:
+        - bubblemelon # 1.15 Branch Manager
+        - claurence # 1.15 RT Lead
+        - idealhack # 1.15 Branch Manager Shadow
+        - k8s-release-robot
+        - lachie83 # 1.15 RT Lead Shadow
+        - nikopen # 1.15 RT Lead Shadow
+        - tpepper # 1.15 RT Lead Shadow
+        privacy: closed
+        teams:
+          patch-release-team:
```

Comment:
Is there a reason we're nesting this team? Because `kubernetes-release-managers` is a highly privileged team, I'd prefer we don't nest anything under it, because then it's not immediately clear who has permissions.

Reply:
@cblecker -- patch release was already nested under release managers, so I didn't want to upset the existing config (https://github.com/kubernetes/org/blob/69deb813392ff8b2259764cddc773c0a8fbe7dc4/config/kubernetes/sig-release/teams.yaml#L111-L121), though I agree that we should probably look at if that needs to be nested in future.